### PR TITLE
Omero 5.4.6

### PR DESCRIPTION
**Pull request by jburel**
Prep for coming 5.4.6
provisionally set Thursday as release date

### CODE REVIEW

File: _posts/2017-05-10-omero-5-4-6.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+layout: post
+title: Release of OMERO 5.4.6
+intro-blurb: The OME team is pleased to announce the release of OMERO 5.4.6.
+---
+Today we are releasing OMERO 5.4.6.
+This introduces a significant new subsystem for read-only operation
+with which servers can be configured not to make changes to the database,
+the filesystem, or both. The goal is to permit the horizontal scaling
+of OMERO installations. Information on how to configure a read-only server
+are available at [Read-Only](https://docs.openmicroscopy.org/omero/5.4.6/developers/Server/Clustering.html#read-only)
```
Comment (mtbc) [inline _posts/2017-05-10-omero-5-4-6.md]: is

File: _posts/2017-05-10-omero-5-4-6.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+   * handling of script params
+   * pagination calculations
+   * public user login
+   * browsing to user's data in IE
+* fixed the chosen login ports for OMERO.cli
+
+Developer updates include:
+
+* a new command to set custom physical pixel size using OMERO.cli
+* deprecated Repository::pixels and TinyImportFixture
```
Comment (mtbc) [inline _posts/2017-05-10-omero-5-4-6.md]: also `OMEROImportFixture`

**Comment by jburel** [comment]
Staged at https://snoopycrimecop.github.io/www.openmicroscopy.org/omero/downloads/ and https://snoopycrimecop.github.io/www.openmicroscopy.org/2018/05/10/omero-5-4-6.html

File: _posts/2018-05-10-omero-5-4-6.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+layout: post
+title: Release of OMERO 5.4.6
+intro-blurb: The OME team is pleased to announce the release of OMERO 5.4.6.
+---
+Today we are releasing OMERO 5.4.6.
+This introduces a significant new subsystem for read-only operation
+with which servers can be configured not to make changes to the database,
+the filesystem, or both. The goal is to permit the horizontal scaling
+of OMERO installations. Information on how to configure a read-only server
+is available at [Read-Only](https://docs.openmicroscopy.org/omero/5.4.6/developers/Server/Clustering.html#read-only)
```
Comment (mtbc) [inline _posts/2018-05-10-omero-5-4-6.md]: end with period

File: _posts/2018-05-10-omero-5-4-6.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+* reduced the number of threads used by OMERO.web
+* fixed other bugs in OMERO.web including:
+   * broken History tab
+   * handling of script params
+   * pagination calculations
+   * public user login
+   * browsing to user's data in IE
+* fixed the chosen login ports for OMERO.cli
+
+Developer updates include:
```
Comment (mtbc) [inline _posts/2018-05-10-omero-5-4-6.md]: This heading is rendered nearer to the above list than the below.

**Pull request merged**